src/webSockets.re

```diff
@@ -1,114 +1,112 @@
 module MessageEvent = {
   type t;
-  external data : t => 'a = "" [@@bs.get];
-  external arrayBufferData : t => Js.Typed_array.array_buffer = "data" [@@bs.get];
-  external stringData : t => string = "data" [@@bs.get];
-  external origin : t => string = "" [@@bs.get];
-  external lastEventId : t => string = "" [@@bs.get];
-  external source : t => Js.t {..} = "" [@@bs.get];
-  external ports : t => array (Js.t {..}) = "" [@@bs.get];
+  [@bs.get] external data : t => 'a = "";
+  [@bs.get] external arrayBufferData : t => Js.Typed_array.array_buffer = "data";
+  [@bs.get] external stringData : t => string = "data";
+  [@bs.get] external origin : t => string = "";
+  [@bs.get] external lastEventId : t => string = "";
+  [@bs.get] external source : t => Js.t({.}) = "";
+  [@bs.get] external ports : t => array (Js.t({.})) = "";
```

Comment:
These should be `Js.t({..})`

Reply:
Perhaps I have something setup wrongly, neither `Js.t({..})` nor `jsobj##message` compiles, both show up as syntax errors. Any advice appreciated :D